tests/TransliteratorTest.php

```diff
@@ -46,6 +46,7 @@ public function provideTransliterationCases()
             array('това е тестово заглавие', 'tova-e-testovo-zaglavie'),
             array('это тестовый заголовок', 'eto-testovyi-zagolovok'),
             array('führen Aktivitäten Haglöfs', 'fuhren-aktivitaten-haglofs'),
+            array('that it\'s \'eleven\' \'o\'clock\'', 'that-its-eleven-oclock'),
```

Comment:
you should use double-quotes around the string, to avoid escaping single quotes. It will be more readable